sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -370,10 +405,9 @@ static rom_error_t rom_ext_attestation_silicon(void) {
       kDiceKeyUds.keygen_seed_idx, kDiceKeyUds.type,
       *kDiceKeyUds.keymgr_diversifier));
   hardened_bool_t cert_valid = kHardenedBoolFalse;
-  uint32_t cert_size = 0;
   HARDENED_RETURN_IF_ERROR(cert_x509_asn1_check_serial_number(
```

Comment:
Do we need to check if the "dice_cert_obj" is X509 before parsing it?

Reply:
we will eventually when you add the CWT certgen. For now there is only one cert type though. We can add this check in a follow up that adds the CWT cert gen support.